pkgs/development/ocaml-modules/luv/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildDunePackage, fetchurl, ctypes }:
+
+buildDunePackage rec {
+  pname = "luv";
+  version = "0.5.6";
+  useDune2 = true;
+
+  src = fetchurl {
+    url = "https://github.com/aantron/luv/releases/download/${version}/luv-${version}.tar.gz";
+    sha256 = "119nv250fsadvrs94nwzk7qvlwr0kvcpkbwcmfkh13byg8nhkn1m";
+  };
+
```

Allow configure script to use `file`

```suggestion

  postConfigure = ''
    for f in src/c/vendor/configure/{ltmain.sh,configure}; do
      substituteInPlace "$f" --replace /usr/bin/file file
    done
  '';

  nativeBuildInputs = [ file ];
  
```

---

shouldn't `file` be `${file}/bin/file` instead?

---

It’s in `PATH`, so not necessary.